src/brasil/gov/tiles/tiles/templates/audiogallery.pt

```diff
@@ -6,92 +6,98 @@
+                                <span i18n:translate="">Required Update</span>
+                                <span i18n:translate="">To view content you need Flash update support to a newer version.</span> <a href="http://get.adobe.com/flashplayer/" title="" i18n:translate="">Click here.</a>
                             </div>
                         </div>
-                        <div class="jp-no-solution">
-                            <span i18n:translate="">Required Update</span>
-                            <span i18n:translate="">To view content you need Flash update support to a newer version.</span> <a href="http://get.adobe.com/flashplayer/" title="" i18n:translate="">Click here.</a>
-                        </div>
+                    </div>
+                    <div>
+                        <h4 class="audiogallery-item-title" tal:content="first_item/title"></h4>
                     </div>
                 </div>
-                <div>
-                    <h4 class="audiogallery-item-title" tal:content="first_item/title"></h4>
+                <div class="audiogallery-items-container">
+                    <ul>
+                        <tal:block tal:repeat="item items">
+                            <li tal:attributes="class python:'selected' if repeat['item'].start else ''">
+                                <a class="audiogallery-item" tal:attributes="data-audio-url python:view.get_item_url(item);
+                                                                             href python:view.get_item_url(item);
+                                                                             title item/description"
+                                   tal:content="item/title"></a>
+                            </li>
+                        </tal:block>
+                    </ul>
                 </div>
-            </div>
-            <div class="audiogallery-items-container">
-                <ul>
-                    <tal:block tal:repeat="item items">
-                        <li tal:attributes="class python:'selected' if repeat['item'].start else ''">
-                            <a class="audiogallery-item" tal:attributes="data-audio-url python:view.get_item_url(item);
-                                                                         href python:view.get_item_url(item);
-                                                                         title item/description"
-                               tal:content="item/title"></a>
-                        </li>
-                    </tal:block>
-                </ul>
-            </div>
-            <div class="audiogallery-footer-container">
-                <a class="audiogallery-footer-link" tal:content="view/data/footer_text"
-                   tal:attributes="href collection/absolute_url"></a>
+            </span>
+            <div tal:condition="python:view.data.get('footer_text')" class="audiogallery-footer-container">
```

Comment:
here you are asking 3 times for `footer_text` it would be better to initialize a variable, even as this may be not costly in terms of performance.